avro-builder/builder/src/main/java/com/linkedin/avroutil1/builder/SchemaBuilder.java

```diff
@@ -238,6 +244,11 @@ public static void main(String[] args) throws Exception {
       skipCodegenIfSchemaOnClasspath = Boolean.TRUE.equals(Boolean.parseBoolean(value));
     }
 
+    List<String> jsonPropsToIgnoreInCompare = new ArrayList<>();
```

Comment:
I saw eventually you have this as a HashSet. why not have it as a Set at the beginning?